pkg/model/filters/filters_test.go

```diff
@@ -91,14 +91,14 @@ func TestSplitForReportersMerge(t *testing.T) {
 
 	assert.Len(t, q1, 3)
 	assert.Equal(t, SingleQuery{
-		NewMatch("FlowDirection", `"1"`),
+		NewMatch("FlowDirection", `"0"`),
```

Comment:
can u use `constants.Ingress` so we stay in sync ?

Reply:
sure https://github.com/netobserv/network-observability-console-plugin/pull/366/commits/0d28ee6e7ddff57c02c1d2203b22963c4a538c98